app/commands.py

```diff
@@ -216,13 +216,13 @@ async def albert_answer(ep: EventParser, matrix_client: MatrixClient):
     """
     # user_prompt: str = await ep.hl()
     config = user_configs[ep.sender]
-    user_prompt: str = ep.event.body
```

Comment:
@dtrckd Why removing this type hint? It's quite helpful.
When removing some added code in a PR, can you please explain why in the PR comments?

Reply:
> because we've already type-hint the signature of the functions so we know the type.
Let me disagree here. It takes time to go back and forth the function to check the type, that's why I added it here to help.

> Furthermore, why you type-hint this one but not all the other, it does no seems consistent ? 
I'm OK to add type hints on the other ones to be more consistent, but the idea is to to that progressively as we go by when it helps.